src/ngx_http_lua_socket_tcp.c

```diff
@@ -148,6 +165,7 @@ enum {
 
 
 enum {
+    SOCKET_OP_RESUME_CONN,
```

Comment:
Better add new enum constants to the end so that we do not change the values of those existing enum constants.